packages/smooth_app/lib/pages/offline_data.dart

```diff
@@ -0,0 +1,183 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+import 'package:provider/provider.dart';
+import 'package:smooth_app/database/dao_product.dart';
+import 'package:smooth_app/database/dao_product_list.dart';
+import 'package:smooth_app/database/local_database.dart';
+import 'package:smooth_app/generic_lib/loading_dialog.dart';
+import 'package:smooth_app/pages/preferences/user_preferences_list_tile.dart';
+import 'package:url_launcher/url_launcher.dart';
+
+class OfflineDataScreen extends StatefulWidget {
```

Comment:
Please some coherence: rather call it `Page` instead of `Screen` like in the rest of the app, and have the name of the file reflect its content, like `OfflineDataPage` in `offline_data_page.dart`.